glotaran/builtin/models/kinetic_spectrum/test/test_kinetic_spectrum_model.py

```diff
@@ -1,382 +1,233 @@
-    model = KineticSpectrumModel.from_dict(
-        {
-            "initial_concentration": {
-                "j1": {"compartments": ["s1", "s2", "s3"], "parameters": ["j.1", "j.1", "j.1"]},
-            },
-            "megacomplex": {
-                "mc1": {"k_matrix": ["k1"]},
-            },
-            "k_matrix": {
-                "k1": {
-                    "matrix": {
-                        ("s2", "s1"): "kinetic.1",
-                        ("s3", "s2"): "kinetic.2",
-                        ("s3", "s3"): "kinetic.3",
-                    }
-                }
-            },
-            "irf": {
-                "irf1": {
-                    "type": "spectral-multi-gaussian",
-                    "center": ["irf.center"],
-                    "width": ["irf.width"],
-                },
-            },
-            "dataset": {
-                "dataset1": {
-                    "initial_concentration": "j1",
-                    "irf": "irf1",
-                    "megacomplex": ["mc1"],
-                },
-            },
-        }
-    )
-    sim_model = KineticSpectrumModel.from_dict(
-        {
-            "initial_concentration": {
-                "j1": {"compartments": ["s1", "s2", "s3"], "parameters": ["j.1", "j.1", "j.1"]},
-            },
-            "megacomplex": {
-                "mc1": {"k_matrix": ["k1"]},
-            },
-            "k_matrix": {
-                "k1": {
-                    "matrix": {
-                        ("s1", "s1"): "kinetic.1",
-                        ("s2", "s2"): "kinetic.2",
-                        ("s3", "s3"): "kinetic.3",
-                    }
-                }
-            },
-            "shape": {
-                "sh1": {
-                    "type": "gaussian",
-                    "amplitude": "shapes.amps.1",
-                    "location": "shapes.locs.1",
-                    "width": "shapes.width.1",
-                },
-                "sh2": {
-                    "type": "gaussian",
-                    "amplitude": "shapes.amps.2",
-                    "location": "shapes.locs.2",
-                    "width": "shapes.width.2",
-                },
-                "sh3": {
-                    "type": "gaussian",
-                    "amplitude": "shapes.amps.3",
-                    "location": "shapes.locs.3",
-                    "width": "shapes.width.3",
-                },
-            },
-            "irf": {
-                "irf1": {
-                    "type": "spectral-multi-gaussian",
-                    "center": ["irf.center"],
-                    "width": ["irf.width"],
-                },
-            },
-            "dataset": {
-                "dataset1": {
-                    "initial_concentration": "j1",
-                    "irf": "irf1",
-                    "megacomplex": ["mc1"],
-                    "shape": {"s1": "sh1", "s2": "sh2", "s3": "sh3"},
-                },
-            },
-        }
-    )
+MODEL_3C_BASE = """\
```

Comment:
For the naming pattern to be consistent with the rest of the code the 3C should be spelled out, e.g.  `MODEL_3C_BASE` should be `MODEL_3_COMPONENT_BASE`.

Reply:
Although in general I would agree with increased verbosity, in particular in any function or class names, 
this is only a constant used in supporting classes of unit tests. 

After careful consideration (trying different things for a few hours while refactoring) I found this to be the right balance between verbosity, readability and maintainability. Maybe in the future we find a better way. I'll be sure to smash my fist on the table when I do.